.goreleaser.yaml

```diff
@@ -8,7 +8,7 @@ before:
 builds:
   - 
     main: ./cmd/openfga
-    binary: bin/openfga
```

Comment:
Dumping the binary into the `bin/` directory just leads us down a rabbit whole, because the Playground assets are intended to be relative to the `openfga` binary. So to alleviate the scenario where a developer extracts the pre-built binaries and does `cd bin && ./openfga run` (which will error out because the Playground assets are no longer relative to the binary), this helps avoid that issue.

Reply:
Actually, let me try and rework this altogether so that it's not subject to relative path issues.

Reply:
Do we need to update the Makefile as well? `make build` drops the binary in the bin folder.

Reply:
Ah, yes! Good catch.

Reply:
Ok @craigpastro I've updated it by dropping the `/bin` prefix where applicable.